web/src/css/custom.scss

```diff
@@ -105,6 +105,11 @@ img[alt$="#width600"] {
   fill: none;
 }
 
+/* We do not want font ligatures in code blocks. */
+code, pre {
+  font-variant-ligatures: none;
+}
+
```

Comment:
I think this change is uncontroversial, but I want to point it out here so that there are no surprises.